src/Controls/tests/TestCases.Shared.Tests/UITest.cs

```diff
@@ -260,22 +261,30 @@ public override void TestSetup()
 		byte[] TakeScreenshot()
 		{
 			// Since the Appium screenshot on Mac (unlike Windows) is of the entire screen, not just the app,
-			// we are going to maximize the App before take the screenshot.
-			App.EnterFullScreen();
-
-			// The app might not be ready to take the screenshot.
-			// Wait a little bit to complete the system animation moving the App Window to FullScreen.
-			Thread.Sleep(500);
-
-			byte[] screenshotPngBytes = App.Screenshot() ?? throw new InvalidOperationException("Failed to get screenshot");
-
-			// TODO: After take the screenshot, restore the App Window to the previous state.
-			App.ExitFullScreen();
-
-			// Wait a little bit to complete the system animation moving the App Window to previous state.
-			Thread.Sleep(500);
-
-			return screenshotPngBytes;
+			// we are going to crop the screenshot to the app window bounds, including rounded corners.
+			var windowBounds = App.FindElement(AppiumQuery.ByXPath("//XCUIElementTypeWindow")).GetRect();
+			// var windowBounds = (Rectangle)App.CommandExecutor.Execute("getWindowBounds", new Dictionary<string, object>()).Value!;
```

Comment:
This line should be removed.. I'm just waiting for CI to provide new screenshots, then we can push a single commit with all the screenshots and this line removed.

```suggestion

```